### PR TITLE
Dig [Common Scripts Utility Consolidations]

### PR DESCRIPTION
**Pull request by samuelFain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-5049

## Description
- Community Pack Adoption for Dig script.
- The 'Run Validations' step fails on BA114 error: The original pack folder names cannot be changed.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [x] Yes
       - The script Dig moved to CommonScripts pack.
   - [ ] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Dig/pack_metadata.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
 {
-    "name": "Dig",
+    "name": "Dig (Deprecated)",
     "description": "Run Dig automation to get 'A' and 'PTR' records.",
```
Comment (Shellyber) [inline Packs/Dig/pack_metadata.json:3]: Please add deprecated also to the pack description

File: Packs/CommonScripts/ReleaseNotes/1_10_41.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### New: Dig
+- DNS lookup utility to provide 'A' and 'PTR' record  (Available from Cortex XSOAR 6.0.0).
```
Comment (Shellyber) [inline Packs/CommonScripts/ReleaseNotes/1_10_41.md:4]: Add here also the release note for the docker image update

**Review by Shellyber — COMMENTED** [review]
Nice one 👍

File: Packs/Dig/pack_metadata.json (hunk 1 above)
Comment (samuelFain) [reply Packs/Dig/pack_metadata.json:3]: Fixed.

File: Packs/CommonScripts/ReleaseNotes/1_10_41.md (hunk 2 above)
Comment (samuelFain) [reply Packs/CommonScripts/ReleaseNotes/1_10_41.md:4]: Fixed.

**Comment by JasBeilin** [comment]
Please fix the release notes (misspelled word). Great job.

**Review by JasBeilin — APPROVED** [review]

**Pull request merged**